packages/jaspr/lib/src/browser/options.dart

```diff
@@ -0,0 +1,52 @@
+import 'dart:async';
+
+import '../../jaspr.dart';
+
+/// Main class for initializing the Jaspr framework on the client.
+///
+/// Call [Jaspr.initializeApp()] at the start of your app, before any calls to [runApp].
+class Jaspr {
+  static void initializeApp({
+    ClientOptions options = const ClientOptions(),
+  }) {
+    _options = options;
+    options.initialize?.call();
+  }
+
+  static bool get isInitialized => _options != null;
+
+  static ClientOptions get options => _options ?? ClientOptions();
```

Comment:
Since I saw you made the class constant below:

```suggestion
  static ClientOptions get options => _options ?? const ClientOptions();
```